app/tests/conftest.py

```
@@ -0,0 +1,25 @@
+KeyError になることを防ぐ。
+"""
+# Flask 3 で削除された locked_cached_property の互換（flask-babel が参照する）。
+# Flask 3 では flask.helpers.cached_property も削除されているため werkzeug から取得する。
+import flask.helpers
+
+if not hasattr(flask.helpers, "locked_cached_property"):
+    from werkzeug.utils import cached_property
+
+    flask.helpers.locked_cached_property = cached_property
+
+import app.context_processors  # noqa: F401 - patch が app.context_processors を参照する前にモジュールを読み込む
```

Ruff (0.15.1)

[warning] Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)